test_files/jsdoc/diagnostics.txt.transformer

```diff
@@ -0,0 +1,20 @@
+Warning at test_files/jsdoc/jsdoc.ts:44:3: unhandled anonymous type
```

Comment:
@evmar I don't know what's going on in here, but AFAICT this was the previous behaviour, the previous test runner just swallowed these. It looks a bit as if we re-check the comments after manipulating them to include types ourselves.